Quantum-Control-Applications/Superconducting/Single Fixed Transmon/rb.py

```diff
@@ -130,23 +130,19 @@ def play_sequence(sequence_list, depth):
         with for_(depth, 1, depth <= max_circuit_depth, depth + delta_depth):
             with for_(n, 0, n < n_avgs, n + 1):
                 assign(saved_gate, sequence_list[depth])
-                assign(sequence_list[depth], inv_gate_list[depth - 1])
-                wait(5000, "qubit")  # Several T1s, can replace with active reset
+                assign(sequence_list[depth], inv_gate_list[depth - 1]) #making sure to have the last gate as the inverse gate
```

Comment:
You didn't format (black) :)
Once you will, you will see that this takes several lines, so we probably want to move the comment up a line.
Also, Captilize.